app/src/lib/utils/logger.ts

```diff
@@ -6,7 +6,7 @@ const loggerOptions = {
 };
 
 const developmentOptions =
-	process.env.NODE_ENV === 'production'
+	env.NODE_ENV !== 'production'
```

Comment:
je crois que c'est une erreur
```suggestion
	env.NODE_ENV === 'production'
```

Reply:
oups, un accident de test, bien vu